src/globus_cli/parsing/param_types/notify_param.py

```diff
@@ -0,0 +1,63 @@
+import click
+
+
+class NotificationParamType(click.ParamType):
+    def get_metavar(self, param) -> str:
+        return "{on,off,succeeded,failed,inactive}"
+
+    def convert(self, value, param, ctx):
+        """
+        Parse --notify
+        - "" is the same as "off"
+        - parse by lowercase, comma-split, strip spaces
+        - "off,x" is invalid for any x
+        - "on,x" is valid for any valid x (other than "off")
+        - "failed", "succeeded", "inactive" are normal vals
+
+        In code, produces True, False, or a set
```

Comment:
This comment doesn't appear to match current code behavior.

Reply:
This was a good catch, thank you! I should have it nicely fixed, but please let me know if it should be reworded at all.